obspy/core/util/obspy_types.py

```diff
@@ -411,6 +411,14 @@ def imag(self):
                                       upper_uncertainty=_upper)
 
 
+class ObsPyException(Exception):
+    pass
+
+
+class ZeroSamplingRate(ObsPyException):
+    pass
+
+
```

Comment:
Should maybe go in a new `obspy.core.exceptions` or even at a top level `obspy.exceptions` as people will import this if do this across the code base. Otherwise I am all for this change :-)